backend/danswer/document_index/vespa/index.py

```diff
@@ -640,7 +637,6 @@ def semantic_retrieval(
             # needed for highlighting while the N-gram highlighting is broken /
             # not working as desired
             + f'or ({{defaultIndex: "{CONTENT_SUMMARY}"}}userInput(@query)))'
-            + _build_vespa_limit(num_to_retrieve)
```

Comment:
As far as testing shows, adding it to yql vs parameters is the same, feels easier/cleaner to just have it in the params